components/pipeline-service/staging/stone-stage-p01/resources/tekton-results-database-secret-path.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 - op: add
   path: /spec/dataFrom/0/extract/key
-  value: integrations-output/terraform-resources/app-sre-stage-01/stone-stage-p01/redhat-staging-plnsvc-rds
+  value: integrations-output/terraform-resources/appsres05ue1/stone-stage-p01/redhat-staging-plnsvc-rds
```

Comment:
It should be `stone-stage-p01-plnsvc-rds` instead of `redhat-staging-plnsvc-rds`